best-practices/check_deprecated_apis.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-deprecated-apis
+  annotations:
+    policies.kyverno.io/category: Best Practices
+    policies.kyverno.io/description: >-
+      Kubernetes APIs are sometimes deprecated and removed after a few releases. 
+      As a best practice older API versions should be replaced with newer versions.
```

Comment:
Would suggest adding to the description annotation something to the effect of: "This policy validates that no resources using any of these deprecated APIs are created."